etna/search/views.py

```diff
@@ -301,9 +299,14 @@ def get_datalayer_data(self, request: HttpRequest) -> Dict[str, Any]:
 
         custom_dimension9 = self.form.cleaned_data.get("q") or "*"
 
+        result_count = self.get_result_count()
+        if result_count > 10000:
+            result_count = 10001
```

Comment:
```suggestion
```
issue: As per issue related to `customMetric1` with consistency for this PR its agreed to show `customMetric1=10000`  for values `gte 10000` , otherwise actual value. Also mentioned here [DF-334](https://national-archives.atlassian.net/browse/DF-334)

Reply:
Resolved in [a0ccd87](https://github.com/nationalarchives/ds-wagtail/pull/422/commits/a0ccd87f3aca7f9f87788eaccc56a67f9f38fdb9)